### ISSUE TITLE
Melpa release

### ISSUE THREAD

**Issue by danielpza** [emacs-vs/cognitive-complexity#1]
Do you plan to push this package to melpa/elpa/other repositories?

Looks cool, just might be difficult to use if not using straight.

Cheers.

**Comment by jcs090218** [comment]
Yes, but it will take a while since I've other packages in queue.

Now, it's on [JCS-ELPA](https://github.com/jcs-emacs/jcs-elpa).
